src/main/java/com/itechart/javalab/library/service/impl/DefaultBookService.java

```diff
@@ -32,12 +33,20 @@ public static BookService getInstance() {
 
 
     @Override
-    public Optional<List<Book>> getAllBooks(Paginator paginator,boolean isFiltered) {
-        return bookDao.getBooks(paginator,isFiltered);
+    public Optional<List<Book>> getAllBooks(Paginator paginator, boolean isAvailableOnly) {
+        BookFilter bookFilter = BookFilter
+                .builder()
+                .isAvailableOnly(isAvailableOnly)
+                .build();
+        return bookDao.getBooks(paginator, bookFilter);
     }
 
     @Override
-    public Optional<Integer> getNumberOfBooksRecords(boolean isFiltered) {
-        return bookDao.getCountOfBooksId(isFiltered);
+    public Optional<Integer> getNumberOfBooksRecords(boolean isAvailableOnly) {
+        BookFilter bookFilter = BookFilter
+                .builder()
+                .isAvailableOnly(isAvailableOnly)
+                .build();
+        return bookDao.getCountOfBooksId(bookFilter);
```

Comment:
why service method name is get**Number**OfBooksRecords, but dao method is get**Count**OfBooksId?